tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/LocaleCompletion.java

```diff
@@ -193,6 +205,13 @@ static LocaleCompletionResponse getLocaleCompletion(final CLDRLocale cldrLocale,
         public int provisional = 0;
         final public String level;
 
+        // The following are more or less debug items
+        public int ignoredLimited = 0;
+        public int ignoredHidden = 0;
+        public int allXpaths = 0;
+        public int statusMissing = 0;
+        public int ignoredOutOfCov = 0;
```

Comment:
it might be worthwhile to make a stronger distinction between debug and non-debug items

is this data included in json, and if so, is it actually used/needed on the front end?

could these be private?

on the one hand, it can be convenient and flexible like this to be public (and maybe included in json), however in the long run there's a risk of cruft, dead code, and unnecessary dependencies